docs/build.md

```diff
@@ -42,6 +63,8 @@ To upload packages, ONLY IF YOU ARE SUPPOSED TO RELEASE YOUR CHANGES:
 
     $ cd building/upload-debs
     $ aklog sipb    # authenticate to the SIPB AFS cell, if necessary
+        # Note that you will need Kerberos tickets
+        # (generate them with kinit) to access AFS.
```

Comment:
Any reason that one of these is after the 'aklog' and one of these is before?

Reply:
No, I'll fix that.